source/georef/georef.h

```diff
@@ -302,7 +305,8 @@ struct GeoRef {
     std::vector<nf::Autocomplete<nt::idx_t>::fl_quality> find_ways(const std::string & str, const int nbmax, const int search_type,std::function<bool(nt::idx_t)> keep_element, const std::set<std::string>& ghostwords) const;
 
 
-    std::vector<Admin*> find_admins(const type::GeographicalCoord&) const;
+    std::vector<Admin*> find_admins(const type::GeographicalCoord&);
```

Comment:
I don't like such a method not const on GeoRef, that can break everything in the context of multithreading.

I'd prefer to not have this method, as it should only be used during binarisation.